Code/Database/AppveyorScriptDatabase.bat

```diff
@@ -13,3 +13,5 @@ SET PASSWORD="Password12!"
 SET INPUT=%cd%\SQLCreateQuery.sql
 
 %SQLCMD% -S%SERVER% -d%DB% -U%LOGIN% -P%PASSWORD% -i%INPUT% -b
```

Comment:
DeleteMaterial in MaterialDa_Code does two actions (not saying it necasaraly wrong, but i thought da layer should be basic af)